i18n/fr.json

```diff
@@ -1,19 +1,343 @@
+            "invalid_website_url": "URL de site web invalide. Veuillez fournir une URL valide.",
+            "unsupported_format": "Format non pris en charge",
+            "accessibility": {
+                "closed_captions": "Sous-titres",
+                "screen_reader_compatible": "Compatible avec lecteur d'écran"
+            },
+            "pricing": {
+                "free": "Gratuit",
+                "freemium": "Freemium",
+                "paid": "Payant"
+            }
+        },
+        "resources_heading": "Ressources",
+        "resources_heading_hl": "pour les étudiants, par les étudiants",
+        "resources_blurb": "Notre équipe académique sélectionne activement des ressources gratuites pour soutenir les étudiants en génie logiciel dans leurs études.",
+        "search_placeholder": "Rechercher par titre, catégorie ou cours",
+        "contributing_heading": "Intéressé à contribuer ou à demander des ressources ?",
+        "btn_join_discord": "Rejoignez notre Discord",
+        "education_quote_1": "La véritable éducation naît là où",
+        "education_quote_1_hl": "la connaissance rencontre l'accessibilité.",
+        "education_quote_2": "L'équipe académique de l'AÉGL est fière de se tenir à cette intersection",
+        "academic_lead": "Responsable académique",
+        "btn_join_team": "Rejoignez notre équipe",
+        "btn_meet_the_team": "Rencontrer l'équipe",
+        "avg_helpfulness": "{percentage}% d'utilité moyenne",
+        "students_helped": "{students} étudiants aidés",
+        "view_label": "Vue",
+        "view_grid": "Grille",
+        "view_row": "Liste",
+        "n_rows": "Lignes",
+        "filter_label": "Filtrer",
+        "filters_active_label": "Filtres actifs",
+        "filter_academic": "Académique",
+        "filter_career": "Carrière",
+        "filter_technical": "Technique",
+        "filter_video": "Vidéo",
+        "filter_textbook": "Manuel",
+        "filter_website": "Site web",
+        "filter_blog": "Blog",
+        "filter_article": "Article",
+        "filter_english": "Anglais",
+        "filter_french": "Français",
+        "filter_bilingual": "Bilingue",
+        "filter_tier_s": "Niveau S",
+        "filter_tier_a": "Niveau A",
+        "filter_tier_b": "Niveau B",
+        "filter_tier_c": "Niveau C",
+        "filter_placeholder_course": "Sélectionner un cours",
+        "filter_placeholder_category": "Sélectionner une catégorie",
+        "filter_placeholder_format": "Sélectionner un format",
+        "filter_placeholder_language": "Sélectionner une langue",
+        "filter_placeholder_tier": "Sélectionner un niveau",
+        "clear_filters": "Effacer les filtres",
+        "clear_all": "Tout effacer",
+        "sort_label": "Trier",
+        "sort_item_relevance": "Pertinence",
+        "sort_item_alphabetical": "Alphabétique",
+        "sort_tier_worst_best": "Niveau (Du pire au meilleur)",
+        "sort_tier_best_worst": "Niveau (Du meilleur au pire)",
```

Comment:
Again--if these sorts are for resources (f), should these adjectives be feminine?

Reply:
I guess the question is whether "best" and "worst" refer to the level, or the resource itself. I would assume the latter